src/styles/grid.scss

```diff
@@ -0,0 +1,19 @@
+.cards {
+  display: grid;
+  grid-template-columns: 200px;
+  gap: 46px 48px; // вертикальний відступ 48px, горизонтальний 46px
```

Comment:
The comment format should be consistent with the code style rules. Consider using a consistent comment style throughout the file.